test/common/libs/shops.js

```diff
@@ -0,0 +1,72 @@
+import shared from '../../../common';
+import {
+  generateUser,
+} from '../../helpers/common.helper';
+
+describe('shops', () => {
+  let user = generateUser();
+
+  describe('market', () => {
+    let shopCategories = shared.shops.getMarketCategories(user);
+    it('contains at least the 3 default categories', () => {
+      expect(shopCategories.length).to.be.greaterThan(2);
+    });
+    it('does not contain an empty category', () => {
+      _.each(shopCategories, (category) => {
+        expect(category.items.length).to.be.greaterThan(0);
+      });
+    });
+    it('items contain required fields', () => {
+      _.each(shopCategories, (category) => {
+        _.each(category.items, (item) => {
+          expect(item).to.have.keys(['key', 'text', 'notes', 'value', 'currency', 'locked', 'purchaseType', 'class']);
```

Comment:
If these are required keys for each shop, they can be extracted into a variable at the top of the file.